app/plugins/prometheus.md

```diff
@@ -0,0 +1,183 @@
+---
+id: page-plugin
+title: Plugins - Prometheus
+header_title: Prometheus
+header_icon: https://konghq.com/wp-content/uploads/2018/06/prometheus.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Usage
+    items:
+      - label: Log Format
```

Comment:
This label doesn't make sense here since this plugin doesn't log anything.

Instead, we could use `Metrics`.

Reply:
Turns out this has to be the same text as the headings. It is used as a link in the sidebar.